.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,5 +2,16 @@
 
 *Description of changes:*
 
+----
+*Testing*  
+- [ ] Did you run our unit tests? `make test`
+- [ ] Did you run our integration tests? `make integ-test`
+- [ ] Did you add new unit test(s)?
+- [ ] Did you add new integration test(s)?
+----
+*Documentation*  
```

Comment:
Add check for "Updated README if applicable"

Reply:
Done